tests/commandline/CMakeLists.txt

```diff
@@ -1,6 +1,8 @@
 cmake_minimum_required(VERSION 3.5)
 
-if(MSVC)
+if (MINGW)
+  add_custom_target(csdtests ${Python_EXECUTABLE} ${CMAKE_CURRENT_SOURCE_DIR}/test.py --use-wine=true --csound-executable=${CMAKE_BINARY_DIR}/csound --opcode6dir64=${CMAKE_BINARY_DIR}/${CMAKE_BUILD_TYPE} --source-dir=${CMAKE_CURRENT_SOURCE_DIR})
```

Comment:
Seems like it should work to revert the --use-wine change and then pass in using --csound-executable="wine ${CMAKE_BINARY_DIR}/csound". @nonameentername have you tried this?

Reply:
I tried using csound-executable with wine but it did not work.  It treats the space as part of the name of the executable.

Reply:
I think this will be fine then for the PR and we can refactor another time.